packages/terra-application-docs/package.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "@cerner/terra-application-docs",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Contains documentation for packages in the terra-application monorepo",
+  "main": "index.js",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-application.git",
+    "directory": "packages/terra-application-docs"
+  },
+  "keywords": [
+    "eslint",
+    "eslint-config",
+    "Cerner",
+    "Terra"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-application/issues"
+  },
+  "files": [
+    "lib",
+    "src"
+  ],
+  "engines": {
+    "node": ">=10.13.0"
+  },
+  "homepage": "https://github.com/cerner/terra-application",
+  "scripts": {
+    "compile": "babel --root-mode upward src --out-dir lib --copy-files",
+    "lint": "eslint --ext .js,.jsx .",
+    "precompile": "rm -rf lib"
+  },
+  "dependencies": {
+    "@cerner/terra-docs": "^1.0.0",
+    "terra-dev-site": "^6.30.0",
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8"
+  },
+  "peerDependencies": {
+    "terra-application": "^1.35.0",
```

nit: Just wanted to match the dev dependency version.

```suggestion
    "terra-application": "^1.48.0",
```